### DISCUSSION TITLE
per axis window-padding-balance

### DISCUSSION THREAD

**Discussion by kate-jiang** [ghostty-org/ghostty#3637] [Issue Triage]
would be cool if we could set window-padding-balance on a per axis basis. for instance, I'd like to have 0 padding top, 8 bottom, and balanced left/right.

**Comment by mitchellh** [comment]
https://ghostty.org/docs/config/reference#window-padding-x and y already support this

**Reply by kate-jiang** [reply]
sorry, maybe I was unclear. attached two screenshots to illustrate:

1) I have 0 y-padding so the tabline is flush with the top of the window, but the left/right padding is uneven. 

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/9b1f901b-76e4-4ea5-a3f5-ae64a7552c9f" />
2) enabled window-padding-balance so the horizontal padding is even, but now the tabline is not flush with the top of the window. 
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/7062fb70-2411-4382-9c71-95fdc42afb43" />



I've tried a bunch of different permutations of the padding settings but it doesn't look like it's possible. ultimately not a huge issue but would be nice to have. thanks!
